sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificatePolicy.java

```diff
@@ -74,12 +74,12 @@
     /**
      * Creation time in UTC.
```

Comment:
This isn't very helpful, actually. For one, I imagine - like .NET - `OffsetDateTime` always tracks in UTC. When you convert for UNIX file time in seconds, it should always be correct. It just makes callers think about converting when, if I assume correctly, they don't need to. In UX studies we saw people wrestle with that. Instead, just say something like:

```java
/**
 * A @{link OffsetDateTime} indicating when the certificate was created.
 */
```
...or something to that effect.

Same for various instances below using `OffsetDateTime`, if you agree.